### DISCUSSION TITLE
JSON Support

### DISCUSSION THREAD

**Discussion by ssharish** [ozanh/ugo#12] [Q&A] [answered]
Hi, first of all, a big HiFi (virtually) on the project.  I was wondering how to parse (marshal, unmarshal) a JSON string passed from Go to uGo.  Could you please elaborate on it?

**Comment by ozanh** [comment]
Hi, @ssharish 😄 thank you.
Currently there is no support for json. But I am planning to add a json module under stdlib, and it will be similar to Go's json package except encode and decode support.
Marshalling to json looks easy but unmarshalling requires considerable amount of effort to implement it from scratch.

**Comment by ozanh** [comment, marked as answer]
`json` module was introduced 8 months ago. Please see the json doc from https://github.com/ozanh/ugo/blob/main/docs/stdlib.md 
Sorry for the poor documentation, I don't have much time to improve it. `json` module was inspired by Go's json package and adapted to uGO.
